src/index.html

```diff
@@ -1,17 +1,152 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" >
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label class="label">
+          Surname:
+          <input 
+            type="text" 
+            name="surname"
+            autocomplete="off" 
+          >
+        </label>
+
+        <label class="label">
+          Name:
+          <input 
+            type="text" 
+            name="name" 
+            autocomplete="off" 
+          >
+        </label>
+
+        <label class="label">
+          How old are You?
+          <input 
+            type="number"
+            name="age" 
+            min="1" 
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="label">
+          Full date of birth: 
+          <input 
+            type="date" name="birthday" 
+          >
+        </label>
+
+        <label class="label">
+          I accept the term of the agreement
+          <input type="checkbox" name="acceptTerm"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="label">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label> 
+
+        <label class="label">
+          Password:
+          <input 
+            type="password"
+            name="password" 
+            minlength="5" 
+            maxlength="9"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="label">
+
+          <label>
```

Comment:
no space between parent and child
```suggestion
        <label class="label">
          <label>
```